homeassistant/components/onkyo/media_player.py

```diff
@@ -126,10 +128,17 @@
 ]
 ISSUE_URL_PLACEHOLDER = "/config/integrations/dashboard/add?domain=onkyo"
 
-type InputLibValue = str | tuple[str, ...]
+type LibValue = str | tuple[str, ...]
 
 
-def _input_lib_cmds(zone: str) -> dict[InputSource, InputLibValue]:
+def _get_single_lib_value(value: LibValue) -> str:
+    if isinstance(value, str):
+        return value
+    return value[0]
+
+
+@cache
```

Comment:
What is cache supposed to do? I don't recognize it from the HA codebase, I think we then usually have a lru cache, but I know too little about this to say if its good or bad

Reply:
This is an LRU cache. The value doesn't change, that's why it makes sense.
It was suggested here: https://github.com/home-assistant/core/pull/129716#discussion_r1828649486